addons/hr_timesheet/report/hr_timesheet_report_view.xml

```diff
@@ -116,38 +127,42 @@
             </field>
         </record>
 
+        <record id="timesheets_analysis_report_pivot_task" model="ir.ui.view">
+            <field name="name">timesheets.analysis.report.pivot</field>
+            <field name="model">timesheets.analysis.report</field>
+            <field name="arch" type="xml">
+                <pivot string="Timesheets Analysis" sample="1" disable_linking="True">
+                    <field name="amount" string="Timesheet Costs"/>
+                    <field name="unit_amount" type="measure" widget="timesheet_uom"/>
+                    <field name="task_id" type="row"/>
+                    <field name="date" interval="month" type="col"/>
+                </pivot>
+            </field>
+        </record>
+
+        <record id="timesheets_analysis_report_graph_task" model="ir.ui.view">
+            <field name="name">timesheets.analysis.report.graph</field>
+            <field name="model">timesheets.analysis.report</field>
+            <field name="arch" type="xml">
+                <graph string="Timesheets" sample="1" js_class="hr_timesheet_graphview" disable_linking="True">
+                    <field name="amount" string="Timesheet Costs"/>
+                    <field name="unit_amount" type="measure" widget="timesheet_uom"/>
+                    <field name="task_id" type="row"/>
+                </graph>
+            </field>
+        </record>
```

Comment:
Can you move those views at the top of this file to have the views definition first and then the actions definition and finally the eventual menus definition (if we need to add a menu)? Thanks in advance! 🙂